index.md

```diff
@@ -1,13 +1,18 @@
 _“Best practice templates and writing instructions for documenting open source software.”_
 
+### Download
+
+* The [0.1 alpha staw-man release of templates](https://github.com/thegooddocsproject/templates/releases/tag/v0.1) was released at the 2019 Write The Docs - Australia conference.
+* The latest development version of templates areavailable from the [github master branch](https://github.com/thegooddocsproject/templates).
```

Comment:
@camerons We need a space between "are available"

Reply:
Otherwise looks okay to merge.

Reply:
Thanks for the feedback, I think I've worked out how to fix this in git, otherwise will fix in a moment.